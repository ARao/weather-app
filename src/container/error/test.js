import React from 'react';
import { shallow } from 'enzyme';
import { Error } from './index';


describe('Error', () => {
  const errorCode = 400;
  let wrapper
  const history = { push: jest.fn() }

  beforeEach(() => {
    wrapper = shallow(<Error history={history} />);
  })

  it('++++ render Error', () => {

    expect(wrapper.length).toEqual(1);
  });
  it('++++ snapshot match', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('++++ should redirect', () => {
    expect(history.push.mock.calls[0]).toEqual(['/']);
  })

});
