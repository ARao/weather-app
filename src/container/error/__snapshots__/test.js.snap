// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error ++++ snapshot match 1`] = `<ErrorView />`;
